### PR TITLE
Update fixture fixes

### PR DESCRIPTION
**Pull request by Grummfy**
This PR was for a fix on #6  But it was not required ;) But because I have made some change to work on it :
- [x] update fixture (and fix the script to make it working)
  - fix unit test to succeed with the udpate of the fixture ;)
  - add missing dependency of the project

### CODE REVIEW

File: tests/fixtures/update-fixtures-from-github [hunk 1]
```diff
@@ -31,12 +33,17 @@ foreach ($fileList as $file) {
     $path = __DIR__ . '/' . $file['name'];
     file_put_contents($path, $contents);
 
-    $drafter->input($path);
+    if ($file['name'] == 'README.md') {
+	    echo "Skipp readme ;)\n";
```
Comment (hendrikmaus) [inline tests/fixtures/update-fixtures-from-github]: Good catch, but you got yourself a typo: `Skipp` -> `Skip`

Comment (Grummfy) [reply tests/fixtures/update-fixtures-from-github]: nice, thanks

**Comment by hendrikmaus** [comment]
Released v0.1.4

**Comment by Grummfy** [comment]
thanks

**Pull request merged**